src/gui/screen_home.cpp

```diff
@@ -163,6 +163,10 @@ int screen_home_event(screen_t *screen, window_t *window, uint8_t event, void *p
         p_window_header_event_clr(&(pw->header), MARLIN_EVT_MediaError);
     }
 
+    if (p_window_header_event_clr(&(pw->header), MARLIN_EVT_MediaRemoved)) {
```

Comment:
Please walk through the history of this file to see if we reordered the checks for various MARLIN_EVT_errors here. I think we did ... I'd like to avoid breaking some other functionality

Reply:
I've checked history around dozen commits back and we didn't changed order of anything, so It should be fine and not break anything.

Reply:
ok